server/src/routes/api/v1/servicesReviewsRouter.js

```diff
@@ -0,0 +1,26 @@
+import express from "express";
+import { Review } from "../../../models/index.js";
+import cleanUserInput from "../../../../services/cleanUserInput.js";
+import { ValidationError } from "objection";
+
+const servicesReviewsRouter = new express.Router({ mergeParams:true })
```

Comment:
We would probably want to name this `serviceReviews` since it is one service object we're referring to and its reviews